### PR TITLE
HCAP-1149: Negative seats remaining

### PR DESCRIPTION
**Pull request by taraepp**
**Bug description:** employers were able to add participants to cohorts when that cohort had no space. It was not desired that they were able to override the cohort size limit

**Steps to reproduce:**
1. Login to employer portal with HA login
2. Hire a participant
3. Click on the last name link and go to Track graduation
4. Select institute and Cohort(with remaining seats 0)
5. Click on Save Changes

**Solution:** display an error when a full cohort has been selected, do not allow to proceed.

**Tested with the following cohorts:**
image
- Full Cohort (with no seats at all): shows error ✅ 
- Roomy Cohort: shows no error, allows to proceed when selected ✅ 
- Single Space: shows no error & allows first participant to proceed, shows error for 2nd participant ✅ 
- Negative Cohort: shows error ✅ 

Error being displayed now is "Cohort is full"

### CODE REVIEW

File: client/package-lock.json [hunk 1]
```diff
@@ -19555,6 +19555,7 @@
       "resolved": "https://registry.npmjs.org/fsevents/-/fsevents-1.2.13.tgz",
       "integrity": "sha512-oWb1Z6mkHIskLzEJ/XWX0srkpkTQ7vaopMQkyaEIoq0fmtFVxOthb8cCxeT+p3ynTdkk/RZwbgG4brR5BeWECw==",
       "dev": true,
+      "hasInstallScript": true,
```
Comment (alva-fresh) [inline client/package-lock.json:19558]: I wonder if changes to `package-lock` need to be part of this PR? 🤔

**Comment by taraepp** [comment]
Definitely not necessary! I'll fix this first thing tomorrow.

**Comment by fwpushan** [comment]
@taraepp No need to provide a bug or ticket description in your PR description just a ticket number with the link is sufficient.

**Review by fwpushan — DISMISSED** [review]
LGTM

**Review by fwpushan — APPROVED** [review]
All good

**Pull request merged**